hazelcast-spring/src/main/resources/hazelcast-spring-4.0.xsd

```diff
@@ -3899,6 +3899,181 @@
         </xs:attribute>
     </xs:complexType>
 
+    <xs:element name="secure-store-substitution-group" abstract="true"/>
+
+    <xs:complexType name="keystore-entry">
```

Comment:
Is this `keystore-entry` used anywhere?

Reply:
Good catch! A leftover from the previous quarter, will remove it.

Reply:
Removed